### PR TITLE
Fix full circle arc generation

### PR DESCRIPTION
**Pull request by cedrickcooke**
Whoops, forgot to degree-ify this earlier 😅 .

### CODE REVIEW

**Review by twyatt — APPROVED** [review]

File: sample/src/jsMain/kotlin/Main.kt [hunk 1]
```diff
@@ -31,7 +31,7 @@ private fun setupPieChart() {
         val input = document.getElementById(id) as HTMLInputElement
         input.min = min.toString()
         input.max = max.toString()
-        input.step = "0.001"
+        input.step = ((max - min) / 1000f).toString()
```
Comment (cedrickcooke) [inline sample/src/jsMain/kotlin/Main.kt:34]: Semi unrelated change. When I went to test on web, I couldn't actually move the slider for end angle close enough to 2 pi to trigger the full circle. This makes the step size variable such that the last step should be exactly max.

**Review by sdonn3 — APPROVED** [review]
